osfs/os.go

```diff
@@ -168,6 +168,42 @@ func (fs *OS) RemoveAll(path string) error {
 	return os.RemoveAll(fullpath)
 }
 
+// Symlink creates newname as a symbolic link to oldname.
+// All parent directories are created.
+func (fs *OS) Symlink(oldname, newname string) error {
```

Comment:
Yep, I just copied them verbatim from the stdlib, they're not even especially good names. `link` and `target` might be better?